### PR TITLE
Update info.plist

### PR DESCRIPTION
**Pull request by erfinbadrian**
Change version of rn apps following xcode settings.

## Changelog

[iOS][Fixed] - Automatically following version on xcode.

### CODE REVIEW

**Comment by kelset** [comment]
can you give for context for why this change is needed? is that how Xcode14 generates new project? is there a changelog that explains why it's needed?

**Comment by erfinbadrian** [comment]
> can you give for context for why this change is needed? is that how Xcode14 generates new project? is there a changelog that explains why it's needed?

You will have problem when you release it to appstore.
if you make it static with version 1 and not following change on xcode version. 
you will get complicated because need to change version on info.plist. 
with this change you just change version on xcode not need change on info.plist.

**Comment by cipolleschi** [comment]
Could you update the changelog properly, please?
The format should be:
```
## Changelog
[iOS][Fixed] - Message
```

Thank you so much!

**Review by cipolleschi — APPROVED** [review]

**Comment by erfinbadrian** [comment]
> Could you update the changelog properly, please? The format should be:
> 
> ```
> ## Changelog
> [iOS][Fixed] - Message
> ```
> 
> Thank you so much!

Edited sir.

Thank you!

**Comment by erfinbadrian** [comment]
Should i change 

```
<key>CFBundleDisplayName</key>
<string>Hello App Display Name</string>

```

to 

```
<key>CFBundleDisplayName</key>
<string>${PRODUCT_NAME}</string>
```

its will help developer changing display name on xcode only no needed to modify on info.plist.

File: template/ios/HelloWorld/Info.plist [hunk 1]
```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+  <string>>$(MARKETING_VERSION)</string>
```
Comment (iBotPeaches) [inline template/ios/HelloWorld/Info.plist:20]: Looks like an extra `>`

Comment (kelset) [reply template/ios/HelloWorld/Info.plist:20]: cc @cipolleschi ☝️ careful when importing

Comment (erfinbadrian) [reply template/ios/HelloWorld/Info.plist:20]: Should we change Display Name automaticcaly following xcode settings? with change to $(PRODUCT_NAME) ?

Comment (cipolleschi) [reply template/ios/HelloWorld/Info.plist:20]: I wonder how's possible that the CI is green if the `info.plist` is malformed... :/

Comment (erfinbadrian) [reply template/ios/HelloWorld/Info.plist:20]: > I wonder how's possible that the CI is green if the `info.plist` is malformed... :/
> 
> > Should we change Display Name automaticcaly following xcode settings? with change to $(PRODUCT_NAME) ?
> 
> Probably yes. It's better to have a single source of truth and to follow the platform's standard.

is our PR is malformed? on my side working fine. please check again sir.
i have test changed `CFBundleDisplayName `to use `$(PRODUT_NAME)` when installing on emulator&real device somehow its still use project name,not following settings on XCODE display name.

Comment (kelset) [reply template/ios/HelloWorld/Info.plist:20]: @erfinbadrian you can see that there is one extra `>` in the change here `  <string>>$(MARKETING_VERSION)`

Comment (erfinbadrian) [reply template/ios/HelloWorld/Info.plist]: > @erfinbadrian you can see that there is one extra `>` in the change here ` <string>>$(MARKETING_VERSION)`

ahh my bad.
i dont see that sorry. I fixed that. Thank you!

**Comment by kelset** [comment]
@erfinbadrian how did you test this change?

I just tried to create a sample app from main (where your commit has landed) and it breaks with
```
The request to open "org.reactjs.native.example.RNTestProject" failed.
Underlying error (domain=FBSOpenApplicationErrorDomain, code=4):
	The operation couldn’t be completed. Application info provider (FBSApplicationLibrary) returned nil for "org.reactjs.native.example.RNTestProject"
	Application info provider (FBSApplicationLibrary) returned nil for "org.reactjs.native.example.RNTestProject"
```

You can get this result by being on main and running `yarn test-e2e-local -t RNTestProject`

---

we might have to revert this @cipolleschi

**Comment by kelset** [comment]
fixed here: https://github.com/facebook/react-native/pull/35012

**Pull request closed without merging**